config/samples/resources/bigqueryconnectionconnection/bigqueryconnectionconnection_v1alpha1_bigqueryconnectionconnection.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Google LLC
```

Comment:
The filename should be `bigqueryconnection_v1alpha1_bigqueryconnectionconnection.yaml`: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.Samples.md#file-naming